### PR TITLE
Update to use twilio-csharp 5.0.0

### PR DESCRIPTION
**Pull request by smendes**

### CODE REVIEW

**Comment by acamino** [comment]
Thanks @smendes. LGTM!

File: TwilioChat.Tests/TwilioChat.Tests.csproj [hunk 1]
```diff
@@ -75,6 +87,10 @@
     <Reference Include="TestStack.FluentMVCTesting.Mvc4">
       <HintPath>..\packages\TestStack.FluentMVCTesting.Mvc4.3.0.0\lib\NET40\TestStack.FluentMVCTesting.Mvc4.dll</HintPath>
     </Reference>
+    <Reference Include="Twilio, Version=0.0.0.0, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>..\packages\Twilio.5.0.0-rca2\lib\net45\Twilio.dll</HintPath>
```
Comment (well1791) [inline TwilioChat.Tests/TwilioChat.Tests.csproj]: @acamino do you think it's ok to change this manually?

File: TwilioChat.Tests/packages.config [hunk 2]
```diff
@@ -5,6 +5,10 @@
   <package id="Microsoft.AspNet.WebPages" version="3.2.3" targetFramework="net45" />
   <package id="Microsoft.Web.Infrastructure" version="1.0.0.0" targetFramework="net45" />
   <package id="Moq" version="4.2.1510.2205" targetFramework="net45" />
+  <package id="Newtonsoft.Json" version="9.0.1" targetFramework="net45" />
   <package id="NUnit" version="3.0.1" targetFramework="net45" />
+  <package id="Portable.BouncyCastle" version="1.8.0" targetFramework="net45" />
+  <package id="Portable.JWT" version="1.0.5" targetFramework="net45" />
   <package id="TestStack.FluentMVCTesting.Mvc4" version="3.0.0" targetFramework="net45" />
+  <package id="Twilio" version="5.0.0-rca2" targetFramework="net45" />
```
Comment (well1791) [inline TwilioChat.Tests/packages.config]: @acamino do you think it's ok to change this manually?

**Review by well1791 — COMMENTED** [review]
@acamino I'm going to work on the gh-pages-next-gen, but for this branch it's ok to do these changes manually?

File: TwilioChat.Tests/packages.config (hunk 2 above)
Comment (acamino) [reply TwilioChat.Tests/packages.config]: Go for it! 😉

**Pull request merged**